packages/esm-patient-vitals-app/src/vitals/vitals-chart.component.tsx

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable prettier/prettier */
```

Comment:
OK I stop the code review here. These linting instructions are there for a reason.